marketplace/commercelayer-products/src/index.js

```diff
@@ -74,10 +41,10 @@ async function fetchSKUs(installationParams, search, pagination) {
     throw new Error(validationError);
   }
 
-  const { clientId, apiEndpoint, clientSecret } = installationParams;
-  const auth = await CLayerAuth.integration({
+  const { clientId, apiEndpoint } = installationParams;
+  const { accessToken } = await CLayerAuth.getIntegrationToken({
```

Comment:
This happens twice. Can you:

- extract to a function
- add a comment mentioning why we pass an empty string, mentioning fixed version of the SDK

Reply:
Done, actually on top of that I make sure we fetch only one token, no need to fetch a new token for every req we issue